pkgpanda/__init__.py

```diff
@@ -356,7 +356,7 @@ def load(self, id: str):
         filename = os.path.join(path, "pkginfo.json")
         try:
             pkginfo = load_json(filename)
-        except FileNotFoundError as ex:
+        except OSError as ex:
```

Comment:
@orsenthil I found other a few other spots where we were catching `FileNotFoundError` with the expectation that it covers all cases where a file doesn't exist on the filesystem and expanded them to cover `OSError`. 88447c58ec8be4e4cc7f1503f28490ad3756de90